AnkiDroid/src/main/java/com/ichi2/libanki/importer/Anki2Importer.kt

```diff
@@ -421,19 +420,19 @@ open class Anki2Importer(col: Collection?, file: String) : Importer(col!!, file)
         val g = src.decks.get(did)
         var name = g.getString("name")
         // if there's a prefix, replace the top level deck
-        if (!TextUtils.isEmpty(mDeckPrefix)) {
+        if (!mDeckPrefix.isNullOrEmpty()) {
             val parts = listOf(*Decks.path(name))
-            val tmpname = TextUtils.join("::", parts.subList(1, parts.size))
-            name = mDeckPrefix!!
-            if (!TextUtils.isEmpty(tmpname)) {
+            val tmpname = parts.subList(1, parts.size).joinToString("::")
+            name = mDeckPrefix
+            if (!tmpname.isEmpty()) {
```

Comment:
The IDE actually suggests this
```suggestion
            if (tmpname.isNotEmpty()) {
```